docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -43,17 +52,17 @@ I have:
 - [ ] Figured if **anything** about the fix warrants tests on Mac/Linux/Windows/Remote/Whatever, and either tested it there if so, or mentioned it in the PR.
 - [ ] Added to or updated docs in this branch, if appropriate
 - [ ] Tests
-     - [ ] Tested the particular change
-     - [ ] Figured if the change might have some side effects and tested those as well.
-     - [ ] Smoke tested the extension as such.
-     - [ ] Tested the VSIX built from the PR (so, after you've submitted the PR). You'll find the artifacts by clicking _Show all checks_ in the CI section of the PR page, and then _Details_ on the `ci/circleci: build` test.
```

Comment:
Did you intend to include whitespace changes?

Reply:
Yes, I was fixing it up according to the markdownlint.